controlpanel/api/cluster.py

```diff
@@ -82,10 +82,10 @@ def __init__(self, app):
     def iam_role_name(self):
         return f"{settings.ENV}_app_{self.app.slug}"
 
-    def create_iam_role(self):
+    def create(self):
```

Comment:
❌ Could we please keep the old names for these two methods? `create()` is very vague and without reading the method body I can't tell what's supposed to happen. (is `App.create()` creating an app as in deploying an app helm chart or is it creating an app pipeline or is it creating an app IAM role or an app or an app group in Auth0 or something else? `create()` is too ambiguous, require thinking and ultimately will lead to confusion and bugs IMHO)

I'd keep `create_iam_role()`/`delete_iam_role()` as it's 100% explicit about the method responsibility and what's going to happen.

Reply:
This method is going to have more things in it than just creating the role - but ok, I'll change the name when I add that extra functionality.